testplan/testing/multitest/result.py

```diff
@@ -99,7 +101,23 @@ def __exit__(self, exc_type, exc_value, tb):
 assertion_state = threading.local()
 
 
-def assertion(func):
+def mark(func: Callable) -> Callable:
```

Comment:
I do not like the name `mark` for this decorator. 
Some ideas:
- `assertion_baseline`
- `assertion_line_target`
- `assertion_helper`
- `grouped_assertion`

Reply:
Would prefer to go with a verb-like originating from mark to make sure it makes sense immediately from user-perspective. What about mark_line_target in this case? Or even, mark_trace_target?

Reply:
Let me try to make this more difficult, lol. How about:

- @report_target
- @trace_line_target
- @mark_line_scope

Reply:
I like `report_target`, combines most of the things we wrote out in different names. @kn-ms what do you think?